src/common/classes/fb_string.h

```diff
@@ -739,6 +739,10 @@ namespace Firebird
 		{
 			return assign(first, last - first);
 		}
+		StringType& assign(const std::string_view& s)
```

Comment:
It's generally recommended to use std::string_view (not const reference) as parameter.